app/views/reports/regions/show.html.erb

```diff
@@ -304,4 +355,7 @@
 <div id="data-json" style="display: none;">
   <%= raw @data.to_json %>
 </div>
-<script src="https://cdn.jsdelivr.net/npm/chart.js@2.8.0"></script>
\ No newline at end of file
+<script src="https://cdn.jsdelivr.net/npm/chart.js@2.8.0"></script>
+<script>
+  window.withLtfu = <%= @with_ltfu %>;
```

Comment:
Is there a better way to pass this flag from the controller to JS?

Reply:
#2039 namespaces this JS code, will address passing this as a parameter to the reports function in that PR.